client/jetpack-cloud/sections/partner-portal/payment-methods/get-stripe-configuration.ts

```diff
@@ -0,0 +1,15 @@
+import { GetStripeConfigurationArgs } from '@automattic/calypso-stripe';
+import wp from 'calypso/lib/wp';
+
+export async function getStripeConfiguration(
+	requestArgs: GetStripeConfigurationArgs & { needs_intent?: boolean }
+) {
+	const config = await wp.req.get(
```

Comment:
Personal preference: this could be directly returned instead of being defined as a constant  💭 

`return await wp.req.get(`

Reply:
Thanks! Updated in 552c1e95e0fe85be9d8cec62d5a6940b6d523367